ESokha/08/Lesson_8_task_1_part_1.html

```diff
@@ -0,0 +1,63 @@
+		<!--<link rel="stylesheet" type="text/css" href="Skeleton/css/skeleton.css">-->
+	</HEAD>
+    <BODY>
+        <div class="container">
+        <div class="row">
+        <div class="ten columns"> <H1>HOW-TO: Программа на Python, Часть 2</H1>
+        </div>
+        <div class="one column"><img src="https://easy-tutorials.kiev.ua/wp-content/uploads/2015/06/Python-logo-notext.svg_Mq7wlhT.png?w=640" alt="Логотип Python" title="Программа на Python часть 1"  width="200" height="200"></div>
+        </div>
+        <div class="row">
+        <div class="five columns">
+        <UL>
+            <LI><a href="Lesson_1_task_3_part_1.html">Программа на Python — часть 1</a></LI>
+            <LI><a href="Lesson_1_task_3_part_2.html">Программа на Python — часть 2</a></LI>
+            <LI><a href="Lesson_1_task_3_part_3.html">Программа на Python — часть 3</a></LI>
+        </UL>
+        </div>
+        <div class="seven columns"><P>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input для получения ответа пользователя, простые типы переменных и простой цикл «<B>for</B>». В этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше программ.</P>
+        </div>
+        </div>
+        <div class="row">
+        <div class="five columns"><H2 id="chap_3">Содержание</H2></div>
+        <div class="three columns"><H2 id="chap_1">Списки</H2></div>
+        <div class="four columns"><h4 id="chap_12"><a href="#chap_3">К содержанию</a></h4></div>
+        </div>
+
+        <div class="row">
+        <div class="five columns"><H3 id="chap_13">HOW-TO: Программа на Python, Часть 2</H3>
+		<UL id="chap_14">
+        <LI><a href="#chap_1">Списки</a></LI>
+        <LI><a href="#chap_2">Комментарии</a></LI>
+        </UL>
+		</div>
+        <div class="seven columns"><P>Давайте рассмотрим другой тип переменных, который называется <Q>списки</Q>. В других языках программирования они называются <Q>массивами</Q>. Если мы обратимся к аналогии с коробками для обуви, то массив (список) представляет собой несколько склеенных друг с другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы запрограммируем его так:</P>
+		<PRE><code>months =
+['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</code></PRE>
+		<P id="chap_15" >Чтобы создать список, мы заключили все значения в квадратные скобки ( &lsquo;[&rsquo; и &lsquo;]&rsquo; ). Мы назвали наш список &lsquo;<q>months</q>&rsquo;. Чтобы использовать список, мы дадим команду print <B>months[0]</B> или <B>months[1]</B> (которые выведут на экран &lsquo;<q>Jan</q>&rsquo; или &lsquo;<q>Feb</q>&rsquo;). Помните: счет начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</P>
+		<PRE><code>print len(months)</code></PRE>
+		</div>
+        </div>
+	</BODY>
+    <!--конец документа-->
+  </HTML> ​
+    <!-- <style> -->
+            <!-- #chap_12{ -->
```

Comment:
inside tag style you should use css comments

Reply:
ok, I will correct